src/components/FormValidator.js

```diff
@@ -55,6 +54,7 @@ class FormValidator{
           this._submitButton.classList.add(this._inactiveButtonSelector)
           this._submitButton.disabled = true;
```

Comment:
```js

          this._submitButton.classList.add(this._inactiveButtonSelector)
          this._submitButton.disabled = true;
```

this is duplicated here and in lines 46-47


Please, define `_disableButton` for that not to duplicate the code




```js
    _disableButton() {
      this._submitButton.classList.add(this._inactiveButtonSelector)
      this._submitButton.disabled = true;
    }
    toggleButtonState(){
      if(!this._getFormValidity(this._inputElements)) {
        this._disableButton()
      } else {

```